src/node_contextify.cc

```diff
@@ -1453,12 +1453,16 @@ static std::vector<std::string_view> throws_only_in_cjs_error_messages = {
     "await is only valid in async functions and "
     "the top level bodies of modules"};
 
-static MaybeLocal<Function> CompileFunctionForCJSLoader(Environment* env,
-                                                        Local<Context> context,
-                                                        Local<String> code,
-                                                        Local<String> filename,
-                                                        bool* cache_rejected,
-                                                        bool is_cjs_scope) {
+// If cached_data is provided, this instead of the on-disk compilation cache
+// from NODE_COMPILE_CACHE would be used.
```

Comment:
```suggestion
// If cached_data is not provided, the on-disk compilation cache
// from NODE_COMPILE_CACHE would be used.
```

Reply:
Hmm, maybe this would be clearer

```suggestion
// If cached_data is provided, the on-disk compilation cache from
// NODE_COMPILE_CACHE (if configured) would be ignored.
```

Because it's rare that `cached_data` is actually provided.

Reply:
Yeah, sounds good to me